src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid.tsx

```diff
@@ -68,14 +68,13 @@ function ResponsiveGrid({
   });
 
   const MARGINS = useMargins ? 8 : 0;
-  // We can't take advantage of isDraggable or isResizable due to performance concerns:
-  // https://github.com/STRML/react-grid-layout/issues/240
```

Comment:
This performance concern mentioned here is the `react-grid-layout` children were unmounted when `isDraggable` and `isResizable` were dynamically changed. This issue has since been resolved with this fix: https://github.com/react-grid-layout/react-grid-layout/pull/892.